src/inverseRobot.js

```diff
@@ -4,10 +4,22 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  const toArray = Object.entries(robot);
+
+  for (const key of toArray) {
+    result[key[1]] = key[0];
+  }
+
+  if (Object.keys(result).length < Object.values(robot).length) {
+    return null;
+  } else {
+    return result;
+  }
```

Comment:
You can iterate over object keys without converting it into array. Also, you can return null as soon as you know that there's already such key.
Your solution works but it might be better. Try to figure out another solution according to my comment.

Reply:
Випадково зробив commit з кривою назвою, вибачаюсь..